### ISSUE TITLE
Issue of the pre-trained checkpoints

### ISSUE THREAD

**Issue by yekeren** [yuweijiang/HGL-pytorch#5]
Dear author, thanks for sharing your work. I tried to reproduce your results on the validation set. However, the checkpoint you provided ([https://drive.google.com/drive/folders/1ux9YG3sRmUVvsCt1nHwlB5Egw43NDsK1?usp=sharing]) only achieves 63.7 accuracies on the validation set, which is the same as the original R2C model. I wonder if you mistakingly uploaded wrong checkpoint files.

Answer acc:    0.637
Rationale acc: 0.705
Joint acc:     0.452

**Comment by yekeren** [comment]
My bad. I made a stupid mistake.

Answer acc:    0.691
Rationale acc: 0.705
Joint acc:     0.490
